test/test_parser.cpp

```diff
@@ -781,6 +780,9 @@ TEST(Parser_LinksToDifferentTable)
     list_2.add(item_keys[2]);
     list_2.add(item_keys[3]);
 
+    verify_query(test_context, t, "items = obj('class_Items', 'coffee')", 0); // nobody buys coffee
+    verify_query(test_context, t, "items = obj('class_Items', 'milk')", 2);   // but milk
```

Comment:
What do you think about the parser also supporting obj syntax without the `class_` prefix? Eg: `obj('Items', 'milk')` That might make it more user friendly if people ever want to manually write these types of queries. Of course, we will always add the class prefix when serializing these to the server.

Reply:
I thought about that, and now that you bring it up, it is probably a good idea.